### DISCUSSION TITLE
The data importer has been polling for more than 601 seconds. It has stopped, to prevent eternal loops.

### DISCUSSION THREAD

**Discussion by mitzone** [firefly-iii/firefly-iii#8980] [Firefly III]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [X] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem
- [X] ... it's [definitely a Firefly III issue, not me](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

Hello,
I receive this message on the latest version. Importing via GoCardless.
Any way to tune the 600 seconds timeout?

I changed maxCheckCount: 3600
here
resources/js/v2/src/pages/conversion/index.js
and here
resources/js/v2/src/pages/submit/index.js

but I receive the same error message. 
Thank you.

### Debug information

I changed maxCheckCount: 3600
here
resources/js/v2/src/pages/conversion/index.js
and here
resources/js/v2/src/pages/submit/index.js

**Comment by JC5** [comment]
If the import really takes this long, it's the best idea to the command line instead. You won't be able to change the timeout.

**Comment by mitzone** [comment]
I have a lot of rules that need to be parsed, maybe this is why. Is this impacting GoCardless? I self host both Firefly III and the importer. I don't mind having the import running for 1 hour, for example. I had to import 5 days batches for monthe of May so I don't hit that limit.

**Comment by JC5** [comment]
Yes, many rules make it slower. Check out the command line options in the documentation 👍

**Comment by mitzone** [comment]
Thanks. What's the impact of the 600 seconds limit? Does it impact the third party (GoCardless). I'd like to just be able to leave it running, regardless of the time it takes. My wife does the import and stuff. If there is REALLY no option to increase that limit, I guess I'll have to learn the CLI. Let me know. :)

**Reply by JC5** [reply]
It's mostly the UI, most browsers become increasingly unreliable after a while. I'm afraid you'll have to use the CLI.

**Comment by ssstain** [comment]
@JC5 I would prefer to be warned, not overruled. If you would have allowed us polling-timeout tuning and have warned us about the side effects, circumstances would be our responsibility.

**Comment by ssstain** [comment]
> I changed maxCheckCount: 3600 here resources/js/v2/src/pages/conversion/index.js and here resources/js/v2/src/pages/submit/index.js

@mitzone 
Seems, you should have done that in `\public\build\assets\` in `index-Dh7tTHJn.js` and `index-N70yVSnT.js` since they are built, while `src\` are yet to be built. Filenames can be different in other versions. But this is not the proper solution... we need an official setting for that.

**Reply by JC5** [reply]
There will not be an official setting for this I'm afraid.
